exo/inference/pytorch/model/utils.py

```diff
@@ -0,0 +1,63 @@
+import torch
+from torch.nn import functional as F
+
+def sample_logits(logits, temperature=1.0, top_k=0, top_p=1.0, alpha_f=0.0, alpha_p=0.0):
```

Comment:
Can this be imported from somewhere rather than copy-pasta into the codebase? It looks like boilerplate code from somewhere.

Reply:
Yes, I was testing it as the default values but will clean that part up. I will set it in the Interface class settings to be used.